cdc-accelerator/feature/smsTemplates.test.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios'
+import SmsTemplates from './smsTemplates.js'
+import { smsExpectedResponse, getSmsExpectedResponseWithMinimumTemplates } from './test.gigyaResponse.sms.js'
+import { expectedGigyaResponseNok, getSiteConfig } from './test.gigyaResponses.js'
+import fs from 'fs'
+import path from 'path'
+import { credentials, apiKey, srcSiteDirectory } from './test.common.js'
+
+jest.mock('fs')
+jest.mock('axios')
+
+describe('Sms templates test suite', () => {
+    let smsTemplates
+
+    beforeEach(() => {
+        jest.clearAllMocks()
+        smsTemplates = new SmsTemplates(credentials)
+        jest.spyOn(smsTemplates, 'generateOtpTemplateFiles').mockImplementation(() => {})
+        jest.spyOn(smsTemplates, 'generateTfaTemplateFiles').mockImplementation(() => {})
+    })
+
+    describe('Init test suite', () => {
+        test('all sms templates files are generated successfully', async () => {
+            axios.mockResolvedValueOnce({ data: smsExpectedResponse })
+            fs.existsSync.mockReturnValue(false)
+
+            await smsTemplates.init(apiKey, getSiteConfig, srcSiteDirectory)
+
+            expect(smsTemplates.generateOtpTemplateFiles).toHaveBeenCalledTimes(1)
```

Comment:
We need to expect that the files, with the right path, are written with the expected content